regression-test/suites/variant_github_events_new_p2/load.groovy

```diff
@@ -128,4 +128,26 @@ suite("regression_test_variant_github_events_p2", "nonConcurrent,p2"){
     sql """DELETE FROM github_events where k >= 9223372036854775107"""
 
     qt_sql_select_count """ select count(*) from github_events_2; """
+
+    trigger_and_wait_compaction("github_events", "full")
+
+    // query and filterd by inverted index
+    profile("test_profile_1") {
+        sql """ set enable_common_expr_pushdown = true; """
+        sql """ set enable_common_expr_pushdown_for_inverted_index = true; """
+        sql """ set enable_pipeline_x_engine = true;"""
```

Comment:
Is enable_pipeline_x_engine necessary? Setting it explicitly will prevent session var fuzzy.

Reply:
profile needed